tracer/src/Datadog.Trace.OpenTracing/Datadog.Trace.OpenTracing.csproj

```diff
@@ -18,7 +18,7 @@
     <PackageReference Include="OpenTracing" Version="0.12.1" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' ">
```

Comment:
It turns out, this entire dependency isn't necessary, so I removed it in https://github.com/DataDog/dd-trace-dotnet/pull/5418